MediaBrowser.XbmcMetadata/Parsers/EpisodeNfoParser.cs

```diff
@@ -91,44 +71,20 @@ protected override void Fetch(MetadataResult<Episode> item, string metadataFile,
                     xml = xmlFile.Substring(0, index + srch.Length);
                     xmlFile = xmlFile.Substring(index + srch.Length);
 
-                    using (var stringReader = new StringReader(xml))
-                    using (var reader = XmlReader.Create(stringReader, settings))
+                    var additionalEpisode = new MetadataResult<Episode>()
+                    {
+                        Item = new Episode()
+                    };
+
+                    // Extract episode details from additional episodedetails block
+                    ReadEpisodeDetailsFromXml(additionalEpisode, xml, settings, cancellationToken);
+
+                    name.Append(" / ").Append(additionalEpisode.Item.Name);
+                    overview.Append(" / ").Append(additionalEpisode.Item.Overview);
```

Comment:
I think these should only be appended if not null or empty
```suggestion
                    if (!string.IsNullOrEmpty(additionalEpisode.Item.Name))
                    {
                        name.Append(" / ").Append(additionalEpisode.Item.Name);
                    }

                    if (!string.IsNullOrEmpty(additionalEpisode.Item.Overview))
                    {
                        overview.Append(" / ").Append(additionalEpisode.Item.Overview);
                    }
```

Reply:
Should they be completely ignored? If the nfo contains three episodes and the second has no title, the combined title would be `Title 1 / Title 3`.
On the other hand this would allow to create nfo files in which a simple (not combined) title can be defined without using the metadata editor (empty except for first entry) and still provide the overview for all entries.

Reply:
I think the end result of `Title 1 / Title 3` is correct (and ideal?)
Sounds like a good candidate for an extra test

Reply:
Since 10.9 this is what our TMDB provider does too: Append title and description, separated by a ` / `

Reply:
I think it makes sense to also combine the <originaltitle> for multi episodes. As a user I would expect that the shown title and original title for an episode match each other (except for the language).
If you agree I, will add it to both the NfoParser and the TMDB provider.

Reply:
TMDB doesn't provide the original title directly and the original language is only available on the series/show level.
I'm not familiar with the code but the only way I see to implement it on the TmdbEpisodeProvider is
- fetch series
- extract original_language
- check if metadata language differ
  - no: OriginalTitle = Title
  - yes: either fetch Episode again with original_language or fetch Translations and search for entry with original_language

Let me know if this is OK or if there is another way to achieve this.